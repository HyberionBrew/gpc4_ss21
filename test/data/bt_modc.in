2: x = 1
5: x = 3
6: x = 9
9: x = 4
12: x = 24
13: x = 32
14: x = 43
17: x = 2
19: x = 23
20: x = 12
22: x = 24
23: x = 32
24: x = 43
27: x = 2
29: x = 23
30: x = 12
