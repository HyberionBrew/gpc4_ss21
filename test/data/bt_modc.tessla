in x : Events[Int]
def y = x % 2
out y
