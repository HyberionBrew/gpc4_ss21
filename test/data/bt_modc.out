2: y = 1
5: y = 1
6: y = 1
9: y = 0
12: y = 0
13: y = 0
14: y = 1
17: y = 0
19: y = 1
20: y = 0
22: y = 0
23: y = 0
24: y = 1
27: y = 0
29: y = 1
30: y = 0
